addon_service/configured_citation_addon/serializers.py

```diff
@@ -0,0 +1,62 @@
+from rest_framework_json_api import serializers
+from rest_framework_json_api.relations import ResourceRelatedField
+from rest_framework_json_api.utils import get_resource_type_from_model
+
+from addon_service.abstract.configured_addon.serializers import (
+    ConfiguredAddonSerializer,
+)
+from addon_service.addon_operation.models import AddonOperationModel
+from addon_service.authorized_citation_account.models import AuthorizedCitationAccount
+from addon_service.common import view_names
+from addon_service.common.serializer_fields import DataclassRelatedLinkField
+from addon_service.configured_citation_addon.models import ConfiguredCitationAddon
+
+
+RESOURCE_TYPE = get_resource_type_from_model(ConfiguredCitationAddon)
+
+
+class ConfiguredCitationAddonSerializer(ConfiguredAddonSerializer):
+    """api serializer for the `ConfiguredCitationAddon` model"""
+
+    root_folder = serializers.CharField(required=False, allow_blank=True)
```

Comment:
I don't think `root_folder` is needed on the `ConfiguredCitationAddonSerializer`. It shouldn't be needed on the `ConfiguredStorageAddonSerializer` as well.

Reply:
we need `root_folder`, as we have to bind our node to certain collection. Or do you imply that this should be declared somewhere else?

Reply:
Yes, we need to bind our node to certain collection. But `root_folder` defined here would imply that all the nodes that are tied to this citation addon share the same `root_folder`. I don't think this is correct. The root folder should be different for each node and that's why it is already on `AuthorizedStorageAccount` model.

Reply:
Ok, I've dug even deeper, and it seems that `ConfiguredCitationAddon` has 1 to 1 relation with Node, so for each Node new `ConfiguredCitationAddon` has to be created. In case with AuthorizedAccount, you are 100% right.